website/docs/d/vpc_endpoint_service.html.markdown

```diff
@@ -41,15 +41,29 @@ data "aws_vpc_endpoint_service" "custome" {
 }
 ```
 
+### Filter
+
+```hcl
+data "aws_vpc_endpoint_service" "test" {
+  filter {
+    name   = "service-name"
+    values = ["some-service"]
+  }
+}
+```
+
 ## Argument Reference
 
 The arguments of this data source act as filters for querying the available VPC endpoint services.
 The given filters must match exactly one VPC endpoint service whose data will be exported as attributes.
 
 * `service` - (Optional) The common name of an AWS service (e.g. `s3`).
 * `service_name` - (Optional) The service name that is specified when creating a VPC endpoint. For AWS services the service name is usually in the form `com.amazonaws.<region>.<service>` (the SageMaker Notebook service is an exception to this rule, the service name is in the form `aws.sagemaker.<region>.notebook`).
+* `filter` - (Optional) One or more name/value pairs to use as filters. There are
+several valid keys, for a full reference, check out
+[describe-vpc-endpoint-services in the AWS CLI reference][1].
```

Comment:
Can you please update this documentation to the longer form configuration block style? e.g. https://github.com/terraform-providers/terraform-provider-aws/pull/12400/files#diff-259bbf21af9ff942736c706b725f4d6dR76-R88 Thank you!